packages/manager/modules/pci/src/components/project/regions-list/regions-list.html

```diff
@@ -32,6 +32,14 @@
                         data-disabled="$ctrl.disabled || $ctrl.constructor.isRegionDisabled(regions)"
                         data-required
                     >
+                        <oui-select-picker-picture
+                            data-ng-if="$ctrl.currentRegion.name === regions[0].microRegion.code"
```

Comment:
What about if regions length > 1 ?

Reply:
We are iterating through regions in the ng-repeat, so if regions.length < 1, this block should not exist.